.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "stage": 0
+  "presets": ["stage-0"]
```

Comment:
Sorry, I’m not that experienced, so I failed to compile it to js. I just installed babel-cli latest version and ran ‘npm publish’. Could you explain how to compile it? I’ll reverse these changes.